test/unit/network_spec.js

```diff
@@ -67,11 +67,15 @@ describe('network', function() {
     // The test is valid for FF only: the XHR has support of the
     // 'moz-chunked-array' response type.
     // TODO enable for other browsers, e.g. when fetch/streams API is supported.
-    var m = /Mozilla\/5.0.*?rv:(\d+).*? Gecko/.exec(userAgent);
-    if (!m || m[1] < 9) {
-      expect(true).toEqual(true);
-      done();
-      return;
+    var isFirefoxWithMozChunkedEncodingSupport =
+      /Mozilla\/5.0.*?rv:(?:\d+).*? Gecko/.test(userAgent);
+    let isFetchWithStreamSupport = (typeof Response !== 'undefined' &&
+      'body' in Response.prototype &&
+      typeof ReadableStream !== 'undefined');
+
+    if (!isFirefoxWithMozChunkedEncodingSupport &&
+      !isFetchWithStreamSupport) {
```

Comment:
Nit: Insufficient indentation on this line, please make sure that the two lines are aligned like below:
```javascript
if (!isFirefoxWithMozChunkedEncodingSupport &&
    !isFetchWithStreamSupport) {
```

Reply:
Yeah Sure I'll do that rightway. Thanks for reviewing .